tgui/.prettierrc.yml

```diff
@@ -1,12 +1,13 @@
 arrowParens: always
-bracketSpacing: true
 endOfLine: lf
+importFormatting: oneline
 jsxBracketSameLine: true
 jsxSingleQuote: false
-printWidth: 80
+printWidth: 72
```

Comment:
you might wonder why this is set to 72. printwidth in prettier is not like eslint, it's not a "rule" but a suggestion. having a printwidth of 80 will still create lines that go over that figure, so i'm tuning it down to 72.

Reply:
eeeh, i think 80 will actually work better, more is always better with typescript

Reply:
it will make lines more than 80, so it's your call
people really need to start breaking their components down so we aren't even reaching that point

Reply:
you should try this locally and tell me what you think. it's seems to be doing well, i've been testing it

Reply:
> having a printwidth of 80 will still create lines that go over that figure

only if it can't fit something under 80, if it was 72, it would have the same issue, but it would more aggressively newline everything which is undesirable.

Reply:
it has been reverted to 80 for the coming commit